src/eth_rpc/mod.rs

```diff
@@ -76,10 +76,7 @@ pub async fn run_server(
         .build(socket_addr.parse::<SocketAddr>()?)
         .await?;
 
-    let addr = server.local_addr()?;
-    let handle = server.start(kakarot_rpc_module);
```

Comment:
What do you think about keeping this line? IMO it's not super clear that `server.start(...)` returns a handle, so keeping the assignment adds readability to the code

Reply:
Yes you are right, just changed